<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QFrame" name="frameOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabOutput">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabOutputAscii">
           <attribute name="title">
            <string>ASCII</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="editOutputAscii">
              <property name="documentTitle">
               <string notr="true"/>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="plainText">
               <string notr="true"/>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
              </property>
              <property name="placeholderText">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabOutputHex">
           <attribute name="title">
            <string>HEX</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="editOutputHex">
              <property name="documentTitle">
               <string notr="true"/>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="plainText">
               <string notr="true"/>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
              </property>
              <property name="placeholderText">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="btnOutputClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="framePinoutSignals">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="WCheckBox" name="checkDSR">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="text">
                <string notr="true">DSR</string>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="WCheckBox" name="checkDCD">
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="text">
                <string notr="true">DCD</string>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="WCheckBox" name="checkRNG">
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="text">
                <string notr="true">RNG</string>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="WCheckBox" name="checkCTS">
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="text">
                <string notr="true">CTS</string>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="framePinoutSignals2">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="WCheckBox" name="checkDTR">
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="text">
                <string notr="true">DTR</string>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="WCheckBox" name="checkRTS">
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="text">
                <string notr="true">RTS</string>
               </property>
               <property name="shortcut">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>38</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frameInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="WTextEdit" name="editInput"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnInputClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMacros">
            <property name="text">
             <string>Macros</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSendFile">
            <property name="text">
             <string>Send File</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>547</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Connect"/>
   <addaction name="action_Settings"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/images/48x48/connect.png</normaloff>:/images/48x48/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/images/48x48/settings.png</normaloff>:/images/48x48/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/images/48x48/about.png</normaloff>:/images/48x48/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">wtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>WCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">wcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabOutput</tabstop>
  <tabstop>editOutputAscii</tabstop>
  <tabstop>editOutputHex</tabstop>
  <tabstop>btnOutputClear</tabstop>
  <tabstop>btnInputClear</tabstop>
  <tabstop>btnMacros</tabstop>
  <tabstop>btnSendFile</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
